api.py

```diff
@@ -0,0 +1,19 @@
+from fastapi import FastAPI
+from models import Bin, Item
+from typing import List, Dict
+
+
+from packer import get_best_bins, create_bins, create_items
+from py3dbp.auxiliary_methods import dump_binlist_to_json
+
+app = FastAPI()
+
+
+@app.post("/packer")
```

Comment:
We are missing README.md update on usage.
We are missing `uvicorn` in `requirements.txt`, becouse I suppose we should run it now like: `uvicorn api:app` ?
We are missing Swagger docs schema description on input (models) : 
![image](https://user-images.githubusercontent.com/22714418/199694625-cd5ccb0c-2ac7-4cec-8a2c-4bfe8f274e51.png)

Look at: https://fastapi.tiangolo.com/tutorial/schema-extra-example/